tools/infrastructure/docker/AGL/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -yq \
+    sudo \
+    git \
+    wget \
+    vim \
```

Comment:
@mkorniichuk remove redundant tools from docker file that should only build SDL

Reply:
@LuxoftAKutsan  fixed within 0e2904f7b83ae3e408736ef42ddbd3d8d833bb68